mantle/kola/harness.go

```diff
@@ -597,6 +618,7 @@ type externalTestMeta struct {
 	Tags            string   `json:"tags,omitempty"`
 	AdditionalDisks []string `json:"additionalDisks,omitempty"`
 	MinMemory       int      `json:"minMemory,omitempty"`
+	NonExclusive    bool     `json:"nonExclusive"`
```

Comment:
I'm trying to place what feels more natural here. I almost feel like having this be `Exclusive` instead of `NonExclusive` would be better even if we have internal functions still using the `nonExclusive` nomenclature (i.e. `runNonExclusiveTests()`)

```suggestion
	Exclusive    bool     `json:"exclusive"`
```

Reply:
I agree that `exclusive` feels a lot more natural. I originally used `nonExclusive` because I wanted to provide consistency between the JSON key and where one would look for the related logic. I think `exclusive` is close enough to `NonExclusive` to provide that consistency, while being less awkward.